app/DevelopmentConfig.hs

```diff
@@ -59,45 +139,47 @@ logMAccessShort host user _ requestLine responseCode _ referer _ =
 -- DATABASE CONNECTION STRINGS
 
 -- | The path to the database file, relative to the project root.
-databasePath :: Text
-databasePath = "db/database.sqlite3"
+databasePath :: IO Text
+databasePath = databasePathValue <$> loadConfig
 
 -- FILE PATH STRINGS
 
 -- | The relative path to the directory with the markdown files rendered for site content.
-markdownPath :: String
-markdownPath = "./"
+markdownPath :: IO String
+markdownPath = markdownPathValue <$> loadConfig
 
 -- | The relative path to the directory that contains all of the graph SVG files.
-graphPath :: String
-graphPath = "./graphs/"
+graphPath :: IO String
+graphPath = graphPathValue <$> loadConfig
 
 -- | The relative path to the directory containing all of the generated CSS files.
-genCssPath :: String
-genCssPath = "./public/style/"
+genCssPath :: IO String
+genCssPath = do genCssPathValue <$> loadConfig
```

Comment:
The `do` shouldn't be necessary here (and similarly for the other definitions below)